data/json/monsters/insect_spider.json

```diff
@@ -1857,5 +1857,347 @@
+    "name": { "str": "giant lady bug" },
+    "description": "A red lady bug covered in black spots about a size of a horse.  Despite cute looks it is a ferocious predator capable of killing swarms of ants and aphids.",
+    "default_faction": "ladybug",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "400 L",
+    "weight": "533 kg",
+    "hp": 160,
+    "speed": 110,
+    "material": [ "iflesh" ],
+    "symbol": "B",
+    "color": "red",
+    "morale": 80,
+    "aggression": 40,
+    "melee_skill": 6,
+    "melee_dice": 2,
+    "melee_dice_sides": 6,
+    "melee_cut": 8,
+    "dodge": 1,
+    "armor_bash": 7,
+    "armor_cut": 12,
+    "armor_bullet": 10,
+    "vision_day": 20,
+    "vision_night": 5,
+    "harvest": "arachnid",
+    "anger_triggers": [  "PLAYER_CLOSE" ],
+    "fear_triggers": [ "HURT" ],
+    "death_function": [ "NORMAL" ],
+    "special_attacks": [
+{
+        "type": "bite",
+        "cooldown": 4,
+        "accuracy": 6,
+        "damage_max_instance": [ { "damage_type": "stab", "amount": 15, "armor_multiplier": 0.6 } ]
+ } 
+    ],
+    "flags": ["POISON", "SEES", "HEARS", "SMELLS", "FLIES" , "PATH_AVOID_FIRE", "PATH_AVOID_FALL" ]
+  },
+    {
+    "id": "mon_grasshopper_small",
+    "type": "MONSTER",
+    "name": { "str": "huge grasshopper" },
+    "description": "A large grasshopper a few times bigger than a regular one.",
+    "copy-from": "mon_grasshopper_giant",
+    "proportional": { "hp": 0.5, "speed": 1.1},
+    "volume": "750 ml",
+    "weight": "1 kg",
+    "upgrades": { "half_life": 7, "into": "mon_mantis_giant" }
+  },
+  {
+    "id": "mon_grasshopper_giant",
+    "type": "MONSTER",
+    "name": { "str": "giant grasshopper", "str_pl": "giant grasshoppers" },
+    "description": "A huge green grasshopper about a size of a big dog with long and strong back legs.  It hops, it hops and never stops.",
+    "default_faction": "insect",
+    "bodytype": "insect",
+    "species": [ "INSECT" ],
+    "volume": "25000 ml",
+    "weight": "33 kg",
+    "hp": 40,
+    "speed": 180,
+    "material": [ "iflesh" ],
+    "symbol": "g",
+    "color": "light_green",
+    "aggression": -10,
+    "morale": 10,
+    "melee_skill": 3,
+    "melee_dice": 1,
+    "melee_dice_sides": 4,
+    "melee_cut": 2,
+    "dodge": 4,
+    "armor_bash": 5,
+    "armor_cut": 10,
+    "vision_day": 20,
+    "vision_night": 20,
+    "harvest": "arachnid",
+    "fear_triggers": [ "FIRE", "PLAYER_CLOSE"],
+    "death_function": [ "NORMAL" ],
+     "special_attacks": [ [ "EAT_FOOD", 60 ], [ "EAT_CROP", 80 ], { "type": "leap", "cooldown": 3, "max_range": 10, "allow_no_target": true } ],
+    "flags": ["POISON", "SEES", "HEARS", "SMELLS", "CLIMBS", "PATH_AVOID_FALL", "PATH_AVOID_FIRE" ]
+  },
+  {
+    "id": "mon_antlion_larva",
+    "type": "MONSTER",
+    "name": { "str": "giant antlion larva" },
+    "description": "This huge grub has powerful mandibles and a hungry look.  It hides in dirt and waits for prey to come clother.",
```

Comment:
```suggestion
    "description": "This huge grub has powerful mandibles and a hungry look.  It hides in the dirt and awaits its prey...",
```

Reply:
The ellipsis will need changing (or removing, which is what I would do) - see the [manual of style](https://github.com/CleverRaven/Cataclysm-DDA/blob/master/doc/MANUAL_OF_STYLE.md).